Docs 7947 disable outbound policies pa (#244)
mulesoft/docs-api-manager: modules/ROOT/pages/policies-policy-types.adoc

endif::[]
:keywords: policy, custom, ootb, offline

> Policies can be default, custom, or automated. Each type of policy has its own scope, management, and usability. You can uniformly apply a policy to all APIs or you can apply policies to select API based on a criteria, which is known as resource-level policies.

Reviewer: "or you can apply policies to select API based on a criteria, which is known as resource-level policies." -> "or you can apply a policy to a selected API based on a specific criteria. The latter feature is known as resource-level policies."

Author: This text is not there anymore in the file. Done! Thanks!

> Policies can be default, custom, or automated. Each type of policy has its own scope, management, and usability. You can uniformly apply a policy to all APIs or you can apply policies to select API based on a criteria, which is known as resource-level policies.

Reviewer 2: Please consider the tech input above and then apply these edits if still appropriate:
```suggestion
You can create default, custom, or automated policies, each with its own scope, management, and usability. You can uniformly apply a policy to all APIs, or you can apply _resource-level policies_ to select APIs based on specified criteria.
```

Author: I have removed and/or reworded this para. Done! Thanks!

* xref:policies-custom-offline-landing-page.adoc[Custom Offline Policies]
* xref:automated-policies-landing-page.adoc[Automated Policies]
* xref:policy-mule3-provided-policies.adoc[Default Policies]

> * xref:policies-policy-level.adoc[Resource Level Policies]

Reviewer 2: ```suggestion
* xref:policies-policy-level.adoc[Resource-Level Policies]
```

Author: Done! Thanks!
